### PR TITLE
Kk/burger menu

### PR DESCRIPTION
**Pull request by Wodnik3z**
Made Burger Menu structure for now

### CODE REVIEW

File: ipz_parkinghunter/lib/Pages/BurgerMenu.dart [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                                      // Onetap underline
+              Navigator.pop(context); // Closes sidebar
+              //TBD
+            },
+            leading: Icon(Icons.home),
+            title: Text("Home Page", style: TextStyle(fontSize: 16)),
+          ),
+          ListTile(
+            onTap: () {
+                                      //
```
Comment (WoryszLk) [inline ipz_parkinghunter/lib/Pages/BurgerMenu.dart:45]: What is it ? //

File: ipz_parkinghunter/lib/Pages/BurgerMenu.dart [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                                      // 
+              Navigator.pop(context); // 
+              //TBD 
+            },
+            leading: Icon(Icons.search),
+            title: Text("Search", style: TextStyle(fontSize: 16)),
+          ),
+          ListTile(
+            onTap: () {
+                                      //
```
Comment (WoryszLk) [inline ipz_parkinghunter/lib/Pages/BurgerMenu.dart:54]: again // ?

File: ipz_parkinghunter/lib/main.dart [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
+import 'package:ipz_parkinghunter/Pages/BurgerMenu.dart';
```
Comment (WoryszLk) [inline ipz_parkinghunter/lib/main.dart:2]: idk if it's necessary

**Review by WoryszLk — CHANGES_REQUESTED** [review]

**Review by Wodnik3z — COMMENTED** [review]
Minor fixes done

**Review by WoryszLk — APPROVED** [review]

**Pull request merged**